.github/actions/auto_gen_bind_pr/action.yaml

```
@@ -20,6 +20,16 @@ runs:
       run: |
         sudo apt update
         sudo apt install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
+    - name: Install cross compiler for android
+      if: inputs.triple == 'aarch64-linux-android'
+      shell: bash
+      run: |
+        # ignoreされているディレクトリで作業
+        mkdir target && cd target
+        curl "https://dl.google.com/android/repository/android-ndk-r25c-linux.zip?hl=ja" > ndk.zip
+        echo A | unzip ndk.zip
```

echo Aは不要かも･･･？　あとrm追加も提案してみます。
```suggestion
        unzip ndk.zip && rm ndk.zip
```
NDKインストールするGithub Actions使っても良いかもと思いました！

---

他のライブラリも含め、Actionsを使うように修正しますね